_/scss/base/_layout.scss

```diff
@@ -16,5 +16,30 @@ section {
 }
 
 .hidden {
-    display: none;
-}
\ No newline at end of file
+    opacity: 0;
+}
+
+.message {
```

Comment:
I htink all .message and input related stuff should be moved into the emoji file to keep them together!